src/interpreter/ByteCode.h

```diff
@@ -2268,7 +2269,7 @@ class OpenLexicalEnvironment : public ByteCode {
         m_endPostion = SIZE_MAX;
     }
 
-    Kind m_kind : 2;
+    Kind m_kind : 4;
```

Comment:
Does `Kind` need 4bits to represent the value?
At the current status, `Kind` has 4 values which could be handled by 2bits.

Reply:
I will add some values when implemting class field value init

Reply:
okay :)